addon/initializers/setup-session.js

```diff
@@ -11,5 +11,7 @@ export default function setupSession(registry) {
     store = 'session-store:test';
     registry.register(store, Ephemeral);
   }
+
+  inject(registry, store, 'cookies', 'service:cookies');
```

Comment:
One down-side of this approach is that you end up injecting `service:cookies` into whatever `session-store` method the application is using, regardless of whether that method required cookies.

I could not find a cleaner way to inject it though. Open to suggestions.